src/LeftNav/LeftNav.less

```diff
@@ -105,15 +127,15 @@
   }
 }
 
-.animateWidth {
-  .timingSlowly;
-  .transition-timing-function(@motionSmooth);
-  transition-propery: width;
+.LeftNav--drawer-anim-enter {
+  width: 0;
+  .animateWidth
 }
 
-.animateOpacity {
-  .delayMoment;
-  .timingPromptly;
-  .transition-timing-function(@motionSnappy);
-  transition-property: opacity;
+.LeftNav--drawer-anim-enter.LeftNav--drawer-anim-enter-active {
+  width: 12rem;
```

Comment:
Super minor but I know we've been trying to use the size macros whenever possible. [`@size_s`](https://github.com/Clever/components/blob/f6793755a4c5f2fb41fe02855cd164aee3c83f96/src/less/sizing.less#L13) would work here